Makefile

```diff
@@ -76,6 +76,10 @@ ifeq ($(ARCH),x86)
         DEFINES		:= -DCONFIG_X86_64
 endif
 
+ifeq ($(ARCH),mips)
+        DEFINES		:= -DCONFIG_MIPS -DCONFIG_SMP  -DCONFIG_WEAK_REORDERING_BEYOND_LLSC
```

Comment:
please, remove `-DCONFIG_SMP  -DCONFIG_WEAK_REORDERING_BEYOND_LLSC` as suggested for earlier patches